automations/data/labels.yml

```diff
@@ -122,6 +122,9 @@ groups:
       - name: code
         description: Concerns the software code in the repository
         emoji: "💻"
+      - name: data
+        description: Concerns the data
+        emoji: "📚"
```

Comment:
Very much bikeshedding but a stack of books reminds me of docs more than data. I was leaning more towards these options:

- 💾 💿 (digital data storage)
- 🗃️ 🗄️ (physical data storage)
- 📈 📊 (data output)

Reply:
Thank you @dhruvkb😊. This is funny, at the first step, i was thinking using this => 📊, and then realized  @AetherUnbound suggests the one I added. Hopefully, it functions as it is expected thanks to @obulat.

Reply:
> Very much bikeshedding but a stack of books reminds me of docs more than data. I was leaning more towards these options:
> 
> - 💾 💿 (digital data storage)
> - 🗃️ 🗄️ (physical data storage)
> - 📈 📊 (data output)

Are you suggesting we should use 2 icons for data, @dhruvkb? Which, if you had to choose?

Reply:
I am ambivalent! Though I do like the cabinets personally 😆 I mostly chose the books because of our "catalog" 😁

Reply:
I was giving ~6 choices without any preference for any of them, although it definitely should be just one.

Reply:
Emoji updated to cabinets 🗄️.